pages/pipelines/hosted_agents/terminal_access.md

```diff
@@ -0,0 +1,32 @@
+# Hosted agents terminal access
+
+The Buildkite hosted agents feature provides you with _terminal/console access_ to jobs running on hosted agents. This feature is useful in allowing you to:
+
+- Understand what components are installed, as you set up your pipeline.
+- Test the behavior of different scripts (because they may not be well-documented).
+- Debug issues that are not reproducible in your local environment.
+
+This can be useful when migrating your pipelines across to [queues](/docs/clusters/manage-queues) on Buildkite hosted agents.
+
+## Activate terminal access on hosted agents
+
+Before you can use the terminal access feature on your Buildkite hosted agents, you'll need to activate this feature for your Buildkite organization, which requires Buildkite organization administrator permissions.
+
+To activate hosted agent terminal access:
+
+1. Select **Settings** in the global navigation to access the [**Organization Settings**](https://buildkite.com/organizations/~/settings) page.
+1. Select **Pipelines** > **Settings** to access your organization's [**Pipeline Settings**](https://buildkite.com/organizations/~/pipeline-settings) page.
+1. Scroll down to the **Hosted Agents SSH** and select the **Enable SSH** button.
+1. Confirm this action by selecting the **Enable Hosted Agents SSH** button.
+
+Terminal access is now granted to all Buildkite hosted agents across all clusters within your Buildkite organization.
+
+Be aware that users require build permissions on pipelines to use the terminal access feature on the pipelines' jobs.
```

Comment:
To use the terminal access feature:
- You must be either an organization admin or a cluster maintainer (in the cluster the job is running within)
- You must have build permissions on pipeline that created the job.

Reply:
Aha, thanks for the additional info about 'cluster maintainers'.

Reply:
This is now covered in both sections of the page.